generators/client/templates/react/src/main/webapp/app/shared/reducers/authentication.ts.ejs

```diff
@@ -146,7 +147,16 @@ export const login = (username, password, rememberMe = false) => async (dispatch
     type: ACTION_TYPES.LOGIN,
     payload: axios.post('api/authentication', data, { headers: { 'Content-Type': 'application/x-www-form-urlencoded' } })
   });
+ <%_ if (enableTranslation) { _%>
+  await dispatch(getSession());
```

Comment:
can you update the reducer tests for this as well please

Reply:
At moment there are no unit tests for login action. It's bit complicated as it has different code flow per authentication type (JWT, UAA etc). To write tests, I need to first generate all possible applications on local :(